src/index.coffee

```diff
@@ -11,6 +11,6 @@ module.exports = (app, options) ->
   if !options or options.enabled isnt false
     autoMigrate = (options and options.migration) or 'auto-update'
     autoMigrate = require './' + autoMigrate
-    autoMigrate(app, options)
+    app.autoMigrateDone = autoMigrate(app, options).then -> debug "Done"
```

Comment:
Because `autoMigrate` is the function that returns the promise when called, not the promise it self. 
When you use from the config, you need access to the promise for `done` of the already invoked `autoMigrate()` so you can't know when to invoke your tests... I'll make another small PR and you tell me what you think.